### ISSUE TITLE
Task - CE Filter: Additional batch-load options [Edited]

### ISSUE THREAD

**Issue by AntWeb-org** [SpeciesFileGroup/taxonworks#3753]
### Feature or enhancement

When I have results from the CE Filter, I would like to be able to Assign/Move more options via the Radial Collecting Event in addition to the Geographic area. Collector names (verbatim or otherwise), dates, verbatim and custom attributes would be very helpful to batch update here.

### Location

Task - Filter Collecting Events

### Screenshot, napkin sketch of interface, or conceptual description

![image](https://github.com/SpeciesFileGroup/taxonworks/assets/74991451/2732217b-b46d-4fea-9687-5f2b2a7bff79)

**Comment by mjy** [comment]
Indeed- we want to deprecate the Match tool as much as possible and move that functionality here.

**Comment by LordFlashmeow** [comment]
For the verbatim fields, it would be great if we could copy the entire `Verbatim` section from the New CE task and make that a slice in the batch radial. We could probably use the same checkbox UI behavior as in #3754 to select which fields are changed/cleared.

**Comment by jlpereira** [comment]
These slices are now available on Radial CE
